.devops/pagopa-deploy-pipelines.yml

```diff
@@ -320,6 +324,8 @@ stages:
               ecommerce_host: 'https://api.platform.pagopa.it'
               api_ecommerce_basepath: '/ecommerce/checkout/v1'
               api_ecommerce_basepath_v2: '/ecommerce/checkout/v2'
+              checkout_auth_service_host: 'https://api.platform.pagopa.it'
```

Comment:
can we reuse `pagopa_apim_host` variable?

Reply:
done in a84b2bd8bea3f307dbd719f14972d178b0c3e879

Reply:
I replace all `ecommerce_host` and `checkout_auth_service_host` with the old one `pagopa_apim_host` that has the same value. I removed the 2 duplicated value and uused one for all apim hosted service. Do you agree?